forem/forem
# Order same reactables together in the list

## What type of PR is this? (check all applicable)


## Description

Orders the same reactables together in the vomit list - users, articles and comments are all grouped together.

## QA Instructions, Screenshots, Recordings

1. Go to abuse reports page
2. See ordered vomits.

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] No, and this is why: _small UI change_
